### DISCUSSION TITLE
Not able to access SOC webpage

### DISCUSSION THREAD

**Discussion by ranjithdoosa** [Security-Onion-Solutions/securityonion#1605] [Unsupported Versions] [answered]
Hi,

I recently installed latest So, Not able to find the exact soc tools  URL in order to access all the tools through SO console Help me how can i access the page, whether do i need to change the settings or allow something?

**Comment by idolaman** [comment]
Did you specify your IP address / subnet in so-allow section, at installation configuration?

**Reply by rayrivera** [reply]
Hello, I am not able to connect to the SOC page with either ip or hostname. I am running the newest version of SecurityOnion and have allowed the ip addresses in so-allow firewall. Using Chrome the webpage comes up and states it's not secure then when I select connect anyways nothing happens. Thanks for any help.

**Reply by dougburks** [reply]
@rayrivera This is an old discussion that has already been marked as answered. Instead of replying to somebody else's discussion, please start a new discussion. Thanks!

**Reply by rayrivera** [reply]
Will do Thanks

**Comment by cm-ops** [comment]
Also, did you select connect to the IP address or hostname of your Security Onion installation during install?

**Comment by m0duspwnens** [comment, marked as answer]
@ranjithdoosa 

The URL can be found by connecting to the manager via SSH. The MOTD will display the SOC URL.

![image](https://user-images.githubusercontent.com/17089008/96920213-de1de980-147a-11eb-9527-80a3f62aa34a.png)

Alternatively, you can also find the URL to access the SOC by running the following command on any of your nodes:

`
salt-call pillar.get global:url_base
`

Either of these methods will either return an IP or URL depending on what you selected on the following screen during setup of your manager.
![image](https://user-images.githubusercontent.com/17089008/96919089-466bcb80-1479-11eb-90bc-f18725217596.png)


Now that you have the URL, we need to allow your workstation(s) to access it. This will need to be done by running [so-allow](https://docs.securityonion.net/en/2.3/so-allow.html?highlight=so-allow) on the manager node, then selecting 'a' to add an analyst, and finally adding your workstation IP or subnet.

**Reply by ranjithdoosa** [reply]
Thanks for the response, i remember DHCP has been selected at the time of installation.
My hostname: nature-virtualbox
do not on what port mange is working hence not able to ssh into the server.
--------------------------------------
My ip config details:
enp0s3    Link encap:Ethernet  HWaddr 08:00:27:90:86:3c  
          inet addr:10.0.2.15  Bcast:10.0.2.255  Mask:255.255.255.0
          inet6 addr: fe80::a00:27ff:fe90:863c/64 Scope:Link
          UP BROADCAST RUNNING PROMISC MULTICAST  MTU:1500  Metric:1
          RX packets:32333 errors:0 dropped:0 overruns:0 frame:0
          TX packets:19838 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000 
          RX bytes:33404080 (33.4 MB)  TX bytes:2741224 (2.7 MB)

--------------------------------------
So allow status:
=========================================================================
UFW Rules
=========================================================================

To                         Action      From
--                         ------      ----
22/tcp                     ALLOW       Anywhere                  
22,443,7734/tcp            ALLOW       10.0.2.0/24               
22,443,7734/tcp            ALLOW       192.168.43.0/24           
22/tcp (v6)                ALLOW       Anywhere (v6)             
--------------------------------------
Nslookup details:

nature@nature-VirtualBox:~$ nslookup nature-VirtualBox
Server:		192.168.43.70
Address:	192.168.43.70#53

** server can't find nature-VirtualBox: NXDOMAIN
--------------------------------------
nature@nature-VirtualBox:~$ sudo salt-call pillar.get global:url_base
[sudo] password for nature: 
[ERROR   ] DNS lookup of 'salt' failed.
[ERROR   ] Master hostname: 'salt' not found. Retrying in 30 seconds

--------------------------------------
Please help me where can i find the Exact URL & IP address & Port number, so that i can directly go into browser and log into the page.

**Reply by dougburks** [reply]
From https://docs.securityonion.net/en/2.3/configuration.html:
> If you use DHCP and your IP address changes, this can cause problems. If you want to use DHCP, make sure that you have a DHCP reservation so that your IP address does not change. Otherwise, use a static IP address to be safe.

**Reply by ranjithdoosa** [reply]
Thanks, I have added static ip address as
192.168.56.100 and added it into so-allow firewall list.

But still i did not understand where can i find the URL of SO Console. Let me know if you want me to run any commands so that you can figure out where i am doing the mistake to find the exact URL.
![16035459198024085533996512602889](https://user-images.githubusercontent.com/45313024/97082890-7efed700-162a-11eb-98d5-a4caf5f501e2.jpg)
![Uploading 16035459782561179505838562952015.jpg…]()



Please guide me how can i access Analyst vm or SOC console

**Reply by dougburks** [reply]
What I was saying is that your Security Onion manager (the box that runs Security Onion Console) needs to have a static IP address. So please build a new Security Onion manager using a static IP address and following the documentation at https://docs.securityonion.net/en/2.3/getting-started.html. Once you have a manager with a static IP address and an appropriate so-allow configuration, then you should be able to connect to that manager via any standard web browser from any machine allowed via so-allow.

**Reply by ranjithdoosa** [reply]
Thanks For quick response, i became fan of SO after researching more on internet about it
 But i am failing to do this part to achieve my goal of acessing so console.

Since am very much less familiar with Linux commands.

Is it possible for you to give me any steps to assign static ip to SO manger?

**Reply by dougburks** [reply]
Please follow the steps in the documentation:
https://docs.securityonion.net/en/2.3/getting-started.html

**Reply by ranjithdoosa** [reply]
Hi,

I am able to assign the Static IP during the network configuration process but When i type the statc ip or Hostname in the browser it directly taking me to localhost webpage.
Kindly guide me how can i achieve to access Securityonion console page.With a proper installation tutorial from sucurity Onion team.

Till now  there is no step by step video to reach to the SO console in the internet, all are old vedios. Since the new upgrade having lot of features i am expecting a complete tutorial vedio from SO members. Please consider this request seriously , in my honest opinion this will benefit for lot of people who wants to try SO.

**Reply by dougburks** [reply]
Have you seen this playlist of Security Onion 2 videos?
https://www.youtube.com/watch?v=Y-nZInToH8s&list=PLljFlTO9rB15aVruK-l7FolRR5LTEGqHm

The first video in that playlist covers the process of setting a static IP address and then connecting to that static IP address.

More videos are coming soon.
